compiler/noirc_frontend/src/ast/expression.rs

```diff
@@ -315,6 +322,7 @@ pub enum Literal {
     Array(ArrayLiteral),
     Bool(bool),
     Integer(FieldElement),
+    NegativeInteger(FieldElement),
```

Comment:
passing comment: Instead of creating a separate variant, you can add a signed flag/enum to Integer instead

Reply:
Done